tests/trainer/test_dataloaders.py

```diff
@@ -446,7 +446,6 @@ def test_dataloaders_with_limit_percent_batches(tmpdir, limit_train_batches, lim
 @pytest.mark.parametrize(
     ["limit_train_batches", "limit_val_batches", "limit_test_batches"], [(0, 0, 0), (1, 2, 3), (1, 2, 1e50)]
 )
-@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
 def test_dataloaders_with_limit_num_batches(tmpdir, limit_train_batches, limit_val_batches, limit_test_batches):
```

Comment:
reviewer: I'm updating this to make use of unittest facilities instead of our internal debugger